src/software/ai/hl/stp/tactic/penalty_kick/penalty_kick_tactic_test.py

```diff
@@ -0,0 +1,77 @@
+import software.python_bindings as tbots_cpp
+from proto.message_translation.tbots_protobuf import create_world_state
+from software.simulated_tests.ball_enters_region import (
+    BallEventuallyEntersRegion,
+    BallAlwaysStaysInRegion,
+)
+from software.simulated_tests.ball_moves_forward import BallAlwaysMovesForward
+from software.simulated_tests.friendly_has_ball_possession import (
+    FriendlyAlwaysHasBallPossession,
+)
+from software.simulated_tests.simulated_test_fixture import (
+    pytest_main,
+)
+from proto.import_all_protos import *
+
+
+def test_penalty_kick_no_goalie(simulated_test_runner):
+    # Setup Robot
+    simulated_test_runner.simulator_proto_unix_io.send_proto(
+        WorldState,
+        create_world_state(
+            yellow_robot_locations=[],
+            blue_robot_locations=[tbots_cpp.Point(-3, 0)],
+            ball_location=tbots_cpp.Point(0, 0),
```

Comment:
penalty mark is 6m from the opponent net (-1.5, 0)

https://robocup-ssl.github.io/ssl-rules/sslrules.html#_penalty_mark